Detectors/MUON/MCH/Simulation/include/MCHSimulation/Response.h

```diff
@@ -23,20 +23,24 @@ namespace mch
 class Response
 {
  public:
-  Response() = default;
-
+  Response(int station = 0): mStation(station) { init(); };
   ~Response() = default;
+
+  void init();
+    
   float getQspreadX() { return mQspreadX; };
   float getQspreadY() { return mQspreadY; };
   float getChargeSat() { return mChargeSat; };
   float getChargeThreshold() { return mChargeThreshold; };
   float etocharge(float edepos);
-  double chargePad(float xmin, float xmax, float ymin, float ymax, int detID, float charge);
-  double response(float charge, int detID);
-  float getAnod(float x, int detID);
+  double chargePad(float xmin, float xmax, float ymin, float ymax, float charge);
+  double response(float charge);
+  float getAnod(float x);
   float chargeCorr();
 
  private:
+  //parameter for station number
+  int mStation = 0;
   //proper parameter in aliroot in AliMUONResponseFactory.cxx
```

Comment:
(might consider removing the "question/todos"-type comments) for the PR) (more relevant location for those would be Jira tickets for instance)

Reply:
Done.